packages/office-ui-fabric-react/src/components/pickers/Pickers.doc.tsx

```diff
@@ -25,6 +25,9 @@ export const PickersPageProps: IDocPageProps = {
     },
   ],
   overview: require<string>('!raw-loader!office-ui-fabric-react/src/components/pickers/docs/PickersOverview.md'),
+    bestPractices: require<
+    string
+  >('!raw-loader!office-ui-fabric-react/src/components/pickers/PeoplePicker/docs/PeoplePickerBestPractices.md'),
```

Comment:
```suggestion
  >('!raw-loader!office-ui-fabric-react/src/components/pickers/docs/PickersBestPractices.md'),
```

Reply:
the current path is correct "packages\office-ui-fabric-react\src\components\pickers\PeoplePicker\docs\PeoplePickerBestPractices.md"

![image](https://user-images.githubusercontent.com/1434956/87993165-adef3500-ca9e-11ea-95ea-040e34b5bba8.png)

Reply:
Will do.